django_rest_passwordreset/views.py

```diff
@@ -60,7 +60,7 @@ def clear_expired_tokens():
     clear_expired(now_minus_expiry_time)
 
 
-def generate_token_for_email(email, user_agent='', ip_address=''):
+def generate_token_for_email(email, user_agent='', ip_address='') -> ResetPasswordToken | None:
```

Comment:
```suggestion
def generate_token_for_email(email, user_agent='', ip_address=''):
```

Reply:
Thanks, just saw the `lint` job failing. Updated in my next commit.